narwhals/dataframe.py

```diff
@@ -890,28 +890,57 @@ def drop(self, *columns: str | Iterable[str]) -> Self:
             *columns: Names of the columns that should be removed from the dataframe.
 
         Examples:
-            >>> import pandas as pd
```

Comment:
this docstring's fine it doesn't need updating

Reply:
Can you give me a little more detail, please? I changed a lot of lines, Should I revert everything to the way it was? And please note, in line 717, you requested for it to be changed from `nw.DataFrame()` to `nw.from_native()`